environment.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Bare minimum setup of the environment to use Vanilla's classes.
+ *
+ * @author Alexandre (DaazKu) Chouinard <alexandre.c@vanillaforums.com>
+ * @copyright 2009-2017 Vanilla Forums Inc.
+ * @license http://www.opensource.org/licenses/gpl-2.0.php GNU GPL v2
+ */
+
+if (PHP_VERSION_ID < 50600) {
+    die('Vanilla requires PHP 5.6 or greater.');
+}
+
+// Define the constants we need to get going.
+if (!defined('APPLICATION')) {
+    define('APPLICATION', 'Vanilla');
+}
+if (!defined('APPLICATION_VERSION')) {
+    define('APPLICATION_VERSION', '2.4.200');
+}
+if (!defined('DS')) {
+    define('DS', DIRECTORY_SEPARATOR);
+}
+if (!defined('PATH_ROOT')) {
+    define('PATH_ROOT', __DIR__);
```

Comment:
From a php perspective this doesn't change anything.

For your webserver, you should use [realpath_root instead of document_root if you use symlinks](http://stackoverflow.com/questions/23737627/php-opcache-reset-symlink-style-deployment/23904770#23904770).